### PR TITLE
Updating theme to support Hugo v0.72.0

### PR DESCRIPTION
**Pull request by mibragimchayev**
fixes: #20

### CODE REVIEW

**Comment by mibragimchayev** [comment]
@felicianotech this is ready for your review

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 exampleSite/themes/hugo-theme-lean-launch-page
+.DS_Store
```
Comment (felicianotech) [inline .gitignore:2]: Can you fix this please? Either remove the line completely (this is a mac thing) or at least fix the line ending so this error goes away.

Comment (felicianotech) [reply .gitignore:2]: The error doesn't show in this view but you if view the "Files Changed" tab you'll see it.

Comment (felicianotech) [reply .gitignore]: Looks good now.

**Comment by mibragimchayev** [comment]
@felicianotech ready for re-review. Could you also test this on your end?

**Pull request merged**